wandb/sdk/wandb_init.py

```diff
@@ -446,18 +448,87 @@ def init(
+            If dict, argparse or absl.flags: will load the key value pairs into
+                the runs config object.
+            If str: will look for a yaml file that includes config parameters and
+                load them into the run's config object.
+        project (str, optional): W&B Project.
+        entity (str, optional): W&B Entity.
+        reinit (bool, optional): Allow multiple calls to init in the same process.
+        tags (list, optional): A list of tags to apply to the run.
+        group (str, optional): A unique string shared by all runs in a given group.
+        name (str, optional): A display name for the run which does not have to be
+            unique.
+        notes (str, optional): A multiline string associated with the run.
+        magic (bool, dict, or str, optional): magic configuration as bool, dict,
+            json string, yaml filename.
+        config_exclude_keys (list, optional): string keys to exclude storing in W&B
+            when specifying config.
+        config_include_keys (list, optional): string keys to include storing in W&B
+            when specifying config.
+        anonymous (str, optional): Can be "allow", "must", or "never". Controls
+            whether anonymous logging is allowed.  Defaults to never.
+        mode (str, optional): Can be "online", "offline" or "disabled". Defaults to
+            online.
+        allow_val_change (bool, optional): allow config values to be changed after
+            setting. Defaults to true in jupyter and false otherwise.
+        resume (bool, str, optional): Sets the resuming behavior. Should be one of:
+            "allow", "must", "never", or "auto". Defaults to "never".
+            Cases:
+            - "auto" (or True): automatically resume the previous run on the same machine.
+                if the previous run crashed, otherwise starts a new run.
+            - "allow": if id is set with init(id="UNIQUE_ID") or WANDB_RUN_ID="UNIQUE_ID"
+                and it is identical to a previous run, wandb will automatically resume the
+                run with the id. Otherwise wandb will start a new run.
+            - "never": if id is set with init(id="UNIQUE_ID") or WANDB_RUN_ID="UNIQUE_ID"
+                and it is identical to a previous run, wandb will crash.
+            - "must": if id is set with init(id="UNIQUE_ID") or WANDB_RUN_ID="UNIQUE_ID"
+                and it is identical to a previous run, wandb will automatically resume the
+                run with the id. Otherwise wandb will crash.
+            See https://docs.wandb.com/library/advanced/resuming for more detail.
+        force (bool, optional): If true, will cause script to crash if user can't or isn't
+            logged in to a wandb server.  If false, will cause script to run in offline
+            modes if user can't or isn't logged in to a wandb server. Defaults to false.
+        sync_tensorboard (bool, optional): Synchronize wandb logs to tensorboard or
```

Comment:
I think "wandb logs from" instead of "wandb logs to" is more clear.  We could also say this saves the tfevent file to wandb as well.

Reply:
agreed